docs/guide/third_party.rst

```diff
@@ -0,0 +1,54 @@
+Third-party plugins
+===================
+This page explains how ``napari-matplotlib`` can be used within third party plugins.
+
+``napari-matplotlib`` provides a ready-to-go widget with a Matplotlib toolbar and figure to third party plugin developers
+This widget is customised to match the theme of the main napari window.
+
+The widget can be found at `napari_matplotlib.base.NapariMPLWidget`.
+This class inherits from ``qtpy.QtWidgets.QWidget``.
+
+It is recommended to use `~napari_matplotlib.base.NapariMPLWidget` inside a new widget, adding it to the layout.
+This means you can add additional elements to your plugin layout alongside the Matplotlib figure.
+Here's a short example:
+
+.. code-block:: python
+
+    from qtpy.QtWidgets import QWidget
+    from napari_matplotlib.base import NapariMPLWidget
+
+    class MyPlugin(QWidget):
+        def __init__(self, napari_viewer: napari.viewer.Viewer, parent=None):
+            super().__init__(parent=parent)
+
+            # Any custom setup for your custom widget
+            ...
```

Comment:
```suggestion
            # ...
```
If these are commented out, does `MyPlugin` work? Be cool to have a genuine working skeleton.

Reply:
`...` is valid Python (it's an [Ellipsis object](https://docs.python.org/3.9/library/constants.html#Ellipsis)), and I've often seen it used to denote a placeholder for other code, so I'd be minded to keep this here?